package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "docker-compose-health-check-action",
+  "version": "1.0.0",
+  "description": "<img src=\"assets/logo.png\" alt=\"Docker Compose Health Check Logo\" width=\"128\">",
+  "main": "index.js",
+  "scripts": {
+    "prepack": "npm install && npm run build",
+    "build": "ncc build src/index.ts -o dist",
+    "test": "jest"
+  },
+  "keywords": [],
+  "author": "jaracogmbh",
+  "license": "MIT",
+  "dependencies": {
+    "@actions/core": "^1.10.1",
+    "@actions/github": "^6.0.0"
+  },
+  "devDependencies": {
+    "@types/jest": "^29.5.12",
```

Comment:
can we use dependabot to keep them up to date

Reply:
done in https://github.com/jaracogmbh/docker-compose-health-check-action/pull/2/commits/6a16bf8d88a23c25b99ab00f50a00c17376c74b1